airflow/cli/commands/db_command.py

```diff
@@ -94,3 +95,26 @@ def shell(args):
 def check(_):
     """Runs a check command that checks if db is available."""
     db.check()
+
+
+@cli_utils.action_cli(check_db=False)
+def clean(args):
+    """Upgrades the metadata database"""
+    print(f"DB: {settings.engine.url!r}")
+    db.upgradedb()
+
+
+# lazily imported by CLI parser for `help` command
+all_tables = sorted(config_dict)
+
+
+@cli_utils.action_cli(check_db=False)
+def cleanup_tables(args):
+    """Purges old records in metastore database"""
```

Comment:
nit: There are different versions of same name "metadata database" (L102), "metastore database" (L113) and "metastore tables" (L1324 of `airflow/cli/cli_parser.py`)